classes/04class/exercises/c04-iac01/mail2kvanitha/class04-keypair.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_key_pair" "class04-ec2-key" {
```

Comment:
wow, great. That was not required for the exercise but well done. 

One tip, the values could be variables or listed in a separate file